src/test/kotlin/com/igorwojda/string/issubstring/solution.kt

```diff
@@ -68,6 +68,18 @@ private object Solution3 {
     }
 }
 
+private object Solution4 {
+    private fun isSubstring(str: String, subStr: String): Boolean {
+        if (subStr.isEmpty()) return false
+
+        return str
+            .windowed(subStr.length)
+            .any {
```

Comment:
Lambda containing one instruction should be formatted in one line:

```koltin
.any { it == subStr }
.toMutableList()
```

Pleas update the code